bool isLeftLegOnline
bool isRightLegOnline
bool isRobotOnGround
bool enableWheel